templates/admin/base/search.tmpl

```diff
@@ -12,6 +12,8 @@
 			<a class="{{if eq .SortType "reversealphabetically"}}active{{end}} item" href="{{$.Link}}?sort=reversealphabetically&q={{$.Keyword}}">{{.i18n.Tr "repo.issues.label.filter_sort.reverse_alphabetically"}}</a>
 			<a class="{{if eq .SortType "recentupdate"}}active{{end}} item" href="{{$.Link}}?sort=recentupdate&q={{$.Keyword}}">{{.i18n.Tr "repo.issues.filter_sort.recentupdate"}}</a>
 			<a class="{{if eq .SortType "leastupdate"}}active{{end}} item" href="{{$.Link}}?sort=leastupdate&q={{$.Keyword}}">{{.i18n.Tr "repo.issues.filter_sort.leastupdate"}}</a>
+			<a class="{{if eq .SortType "recentlogin"}}active{{end}} item" href="{{$.Link}}?sort=recentlogin&q={{$.Keyword}}">{{.i18n.Tr "repo.issues.filter_sort.recentlogin"}}</a>
+			<a class="{{if eq .SortType "recentloginreverse"}}active{{end}} item" href="{{$.Link}}?sort=recentloginreverse&q={{$.Keyword}}">{{.i18n.Tr "repo.issues.filter_sort.recentloginreverse"}}</a>
```

Comment:
Not sure if we should add specific sorters like this to the dropdown, it is used for orgs listing as well.
In general it's a bit weird, that this template is shared by user & org listing, while an almost identical search template `explore/search.tmpl` exists, that actually would benefit from being shared between admin & explore listing..
Opinions?

Reply:
Yes, please share templates if possible, we need to reduce duplication. What you could do is something like we already do for issue lists:

````hbs
{{template "shared/issuelist" mergeinto . "listType" "dashboard"}}
````

This would merge a additional `listType=dashboard` property into the root context `.`, making it available to the sub-template. Keep note that `mergeinto` does clone the context once, so it'll induce a tiny performance hit, thought that should be okay for one-time rendered things like these lists.